sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -3222,6 +3409,72 @@ export class BlockBlobClient extends BlobClient {
     );
   }
 
+  /**
+   * Quick query for a JSON or CSV formated blob.
+   *
+   * Example usage (Node.js):
+   *
+   * ```js
+   * // Query and convert a blob to a string
+   * const queryBlockBlobResponse = await blockBlobClient.query("select * from BlobStorage");
+   * const downloaded = await streamToString(queryBlockBlobResponse.readableStreamBody);
+   * console.log("Query blob content:", downloaded);
```

Comment:
For quick query's result, do we know the max size? And how will user handle the result typically?

Reply:
The query response is a stream like download operation. Query does a filtering per input SQL statement.

Not sure about server limitation of max size in response. But it doesn't impact existing SDK design.